parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -147,6 +147,23 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
+    ORAS_DOWNLOAD_DIR=${1}
+    ORAS_DOWNLOAD_URL=${2}
+    ORAS_VERSION=${3}
+
+    mkdir -p $ORAS_DOWNLOAD_DIR
+
+    echo "Installing Oras version $ORAS_VERSION..."
+    ORAS_TMP=${ORAS_DOWNLOAD_URL##*/} # Use bash builtin ## to remove all chars ("*") up to the final "/"
+    retrycmd_get_tarball 120 5 "$ORAS_DOWNLOAD_DIR/${ORAS_TMP}" ${ORAS_DOWNLOAD_URL} || exit $ERR_ORAS_DOWNLOAD_TIMEOUT
+    tar -zxf oras_${ORAS_VERSION}_*.tar.gz -C $ORAS_DOWNLOAD_DIR/${ORAS_TMP}
+    sudo mv $ORAS_DOWNLOAD_URL/${ORAS_TMP}/oras /usr/local/bin/
+
```

Comment:
can we delete the `"$ORAS_DOWNLOAD_DIR"` after to keep the VHD free of download artifacts

Reply:
@Devinwong 

wc: /opt/oras/downloads/oras_1.2.0_linux_amd64.tar.gz: No such file or directory testKubeBinaries:Error: File size of /opt/oras/downloads/oras_1.2.0_linux_amd64.tar.gz from https://github.com/oras-project/oras/releases/download/v1.2.0/oras_1.2.0_linux_amd64.tar.gz is invalid. Expected file size: 4052022 - downlaoded file size:
Deleting resource group vhd-test-1722036494-2130

VHD build tests fail when we delete the source code. Is there a reason to check for the .tar.gz over checking if the package is installed via like package --version output? 

https://github.com/Azure/AgentBaker/blob/1973b6d78eb92a121cd995053ee3778088453bc9/vhdbuilder/packer/test/linux-vhd-content-test.sh#L129

Reply:
@AlisonB319  There is another check a few lines before. which is:
`# if there is a directory with expected name, we assume it's been downloaded and extracted properly`
So if the binaries have been installed to a desired directory, this test should also pass.
If you package can be validated through some command like --version, feel free to add one.
I didn't redesign this part of validation, just using the existing logics to test the packages.